ledger/acctupdates.go

```diff
@@ -2180,6 +2231,10 @@ func (au *accountUpdates) commitRound(offset uint64, dbRound basics.Round, lookb
 		return
 	}
 
+	if updateStats {
+		stats.DatabaseCommitDuration = time.Duration(time.Now().UnixNano()) - stats.DatabaseCommitDuration
```

Comment:
my intention here was to measure the time of the database commit method. While you can't directly see it here ( i.e. it's hidden inside `Atomic` ), that is the method that can take a bulky amount of time.
In order to measure that, you'll need to subtract the "content" of the Atomic function call, i.e.
```golang
stats.DatabaseCommitDuration = time.Duration(time.Now().UnixNano()) - stats.DatabaseCommitDuration - stats.AccountsWritingDuration - stats.MerkleTrieUpdateDuration - stats.OldAccountPreloadDuration
```